test/event.js

```diff
@@ -94,6 +94,37 @@ QUnit.test( ".delegate() and .undelegate()", function( assert ) {
 	} );
 } );
 
+QUnit.test( "Event aliases", function( assert) {
+	assert.expect( 16 );
```

Comment:
Where does the 16 come from? I counted 14:

* in the first block: 4 warnings & 4 `assert.equal`s
* in the second block: 1 warning & 2 `assert.ok`s
* in the third block: 1 warning & 2 `assert.equal`s

What am I missing?